### PR TITLE
Just typos in code

### PR DESCRIPTION
**Pull request by hunger**
Just an attempt to clean out the list of issues I see when I open 60fps :-)

Sorry, the patch names are not too descriptive.

I find the Code Spell Checker very helpful while coding.

### CODE REVIEW

**Comment by hunger** [comment]
OK, let's split this patch set: These are the code changes. The CI seems to be happy about those!

Let's move the other two patches that fix strings and comments as well as markdown warnings out.

**Comment by hunger** [comment]
@tronical, @ogoffart : Your tests are happy with this, so am I:-) Please review.

File: sixtyfps_compiler/passes/binding_analysis.rs [hunk 1]
```diff
@@ -210,7 +210,7 @@ fn visit_implicit_layout_info_dependencies(
         }
         "Text" => {
             vis(&NamedReference::new(&item, "text"));
-            vis(&NamedReference::new(&item, "font_familly"));
+            vis(&NamedReference::new(&item, "font_family"));
```
Comment (ogoffart) [inline sixtyfps_compiler/passes/binding_analysis.rs:213]: this one is interesting!!  that was a bug!

**Review by ogoffart — APPROVED** [review]

File: sixtyfps_compiler/passes/binding_analysis.rs (hunk 1 above)
Comment (hunger) [reply sixtyfps_compiler/passes/binding_analysis.rs:213]: My first bug fix then! :-)

Comment (tronical) [reply sixtyfps_compiler/passes/binding_analysis.rs:213]: Slow and steady wins the race :-)

btw, did you configure the spell checker in any special way?

Comment (hunger) [reply sixtyfps_compiler/passes/binding_analysis.rs:213]: I did add a ton of words to its ignore/workspace list, yes.

You have quite a few apprevtions and shorthand forms in your codebase that the code spellchecker does not like too much!

Comment (tronical) [reply sixtyfps_compiler/passes/binding_analysis.rs:213]: I'm thinking of adding my `cspell.json` to the git repo simply. Or you could also submit yours if you have one already :-)

**Comment by ogoffart** [comment]
We could have spell check as part of the CI, but i suppose it indeed need quite some white list.

File: sixtyfps_compiler/passes/binding_analysis.rs (hunk 1 above)
Comment (hunger) [reply sixtyfps_compiler/passes/binding_analysis.rs:213]: In my experience the list is manageable -- if you do are comfortable to have really short (<= 3 chars) or really long (real words with camelcase/snakecase) local variables.

**Pull request merged**